web/client/actions/map.js

```diff
@@ -187,6 +191,27 @@ const checkMapChanges = (action, source) => ({
     source
 });
 
+const registerEventListener = (eventName, toolName) => ({
+    type: REGISTER_EVENT_LISTENER,
+    eventName,
+    toolName
+});
+
+const unRegisterEventListener = (eventName, toolName) => ({
+    type: UNREGISTER_EVENT_LISTENER,
+    eventName,
+    toolName
+});
+
+const mouseMoveMapEvent = (position) => ({
```

Comment:
```suggestion
/**
 * Triggered on mouse move. (only if some tool is registered on this event. See `registerEventListener`).
 * @param {object} position the position of the mouse on the map.
 */ 
const mouseMoveMapEvent = (position) => ({
```
**You can quickly describe the position object**